Equinox/ModuleAnimation.h

```diff
@@ -0,0 +1,37 @@
+﻿#ifndef __MODULEANIMATION_H__
+#define __MODULEANIMATION_H__
+
+#include "Module.h"
+#include <map>
+#include <MathGeoLib/include/Math/float3.h>
+
+struct Channel
+{
+	std::string NodeName;
+	std::vector<float3> Positions;
+	std::vector<Quat> Rotations;
+};
+
+struct Animation
+{
+	double Duration = 0.f;
+	std::vector<Channel> Channels;
```

Comment:
Why don't we use pointers? Pointers ensure that we always use the same instance, not a copy